sensu/files/plugins/check-elastic.rb

```diff
@@ -120,6 +139,10 @@ def run_threshold_check
     data = get_count
     count = data['count']
 
+    # Override the check name to be more specific as requested
+    if config[:name] != nil
```

Comment:
It behaves differently in the case of an empty string, so probably not all that useful.